apps/prairielearn/src/pages/instructorFileEditor/instructorFileEditor.html.ts

```diff
@@ -0,0 +1,335 @@
+import { html, joinHtml, unsafeHtml } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { compiledScriptTag, nodeModulesAssetPath } from '../../lib/assets.js';
+import { config } from '../../lib/config.js';
+import { encodePath } from '../../lib/uri-util.js';
+
+export function InstructorFileEditor({ resLocals }: { resLocals: Record<string, any> }) {
+  const { fileEdit, authz_data, course_owners, course, __csrf_token } = resLocals;
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head data-ace-base-path="${nodeModulesAssetPath('ace-builds/src-min-noconflict/')}">
```

Comment:
This is sort of a 🤷, but, elsewhere we've used `<meta>` elements to inject page-wide context like this (e.g. CSRF tokens).

Reply:
Done in 562343bda.